source/speech/speech.py

```diff
@@ -1355,6 +1356,14 @@ def _suppressSpeakTypedCharacters(number: int):
 FIRST_NONCONTROL_CHAR = " "
 
 
+def is_editableControl() -> bool:
```

Comment:
Can this get a more descriptive name and a docstring?
```suggestion
def isFocusEditable() -> bool:
```